packages/components/cascader-panel/src/node.vue

```diff
@@ -131,7 +131,11 @@ const doCheck = (checked: boolean) => {
 
 const doLoad = () => {
   panel.lazyLoad(props.node, () => {
-    if (!isLeaf.value) doExpand()
+    if (!isLeaf.value) {
+      doExpand()
+    } else {
+      handleCheck(true)
+    }
```

Comment:
I think it is possible to avoid auto selected and keep it similar to `tree`.

```suggestion
    if (!isLeaf.value) doExpand()
```